components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilderTest.java

```diff
@@ -304,6 +347,52 @@ public void testGetAbsoluteURL(String protocol, String hostName, int port, Strin
 
         String absoluteUrl = null;
 
+        try {
+            if (MapUtils.isNotEmpty(parameters) && MapUtils.isNotEmpty(fragmentParams)) {
+                absoluteUrl =
+                        ServiceURLBuilder.create().addPath(urlPath).setFragment(fragment).addFragmentParameter("key1",
+                                "fragment").addFragmentParameter("key2", "fragment").addFragmentParameter("key3",
+                                "fragment").addFragmentParameter("key4", "fragment").addParameter("key1", "v")
+                                .addParameter("key2", "v").addParameter("key3", "v").addParameter("key4", "v").build()
+                                .getAbsolutePublicURL();
+            } else if (MapUtils.isNotEmpty(fragmentParams)){
+                absoluteUrl =
+                        ServiceURLBuilder.create().addPath(urlPath).setFragment(fragment).addFragmentParameter("key1",
+                                "fragment").addFragmentParameter("key2", "fragment").addFragmentParameter("key3",
+                                "fragment").addFragmentParameter("key4", "fragment").build().getAbsolutePublicURL();
+            } else if (MapUtils.isNotEmpty(parameters)){
+                absoluteUrl =
+                        ServiceURLBuilder.create().addPath(urlPath).setFragment(fragment).addParameter("key1", "v")
+                                .addParameter("key2", "v").addParameter("key3", "v").addParameter("key4", "v").build()
+                                .getAbsolutePublicURL();
+            } else {
+                absoluteUrl =
+                        ServiceURLBuilder.create().addPath(urlPath).setFragment(fragment).build().getAbsolutePublicURL();
+            }
+
+        } catch (URLBuilderException e) {
+            //Mock behaviour, hence ignored
```

Comment:
use standard comment format add space after // and add fullstop at end.

Reply:
Fixed 0c97e69